CMakeLists.txt

```diff
@@ -70,10 +70,10 @@ option (Seastar_DEPRECATED_OSTREAM_FORMATTERS
   ON)
 
 set (Seastar_API_LEVEL
-  "7"
+  "8"
```

Comment:
> json: API level 8: use noncopyable_function in json_return_type
> Hold a shared_body_writer_type so we can copy json_return_type
> without copying the body_writer function.
> 
> This will allow returning a co_routine lambda as body_writer
> that can be safely held as a lw_shard_ptr<noncopyabe_function>
> in the following patch.
> 

Why does json_return_type need to be copyable?